src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -531,7 +531,7 @@ static JavaType of(Object obj) {
         return JavaType.OBJECT;
     }
 
-    // Retrieve JDBC to use with this Java type. By default we use the static JDBC type
+    // Returns the JDBC type to use with this Java type. By default we use the static JDBC type
     // associated with the Java type, ignoring the JDBC type specified by the application.
     // But this behavior is overridden for certain Java types, like InputStream, which
     // require the JDBC type to be specified externally to be able to distinguish between
```

Comment:
4 lines of comments should be wrapped with `/* */` instead of 4 `//`. The comment is also a little off grammatically. Maybe it should be changed to something along the lines of the following:
>Returns the JDBC type to use with this Java type. We use the static JDBC type associated with the Java type by default, ignoring the JDBC type specified by the applications. This behavior is overridden for certain Java types, such as InputStream, which requires the JDBC type to be specified externally in order to distinguish between...